docs/section5/Rule5-38.md

```diff
@@ -0,0 +1,91 @@
+
+# Envelope - Rule 5-38  
+
+**Rule ID:** 5-38  
+**Rule Description:** Skylight SHGC properties shall match the appropriate requirements in Tables G3.4-1 through G3.4-8 using the value and the applicable skylight percentage.  
+**Rule Assertion:** B-RMR subsurface: SHGC = expected value  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()
+  2. get_opaque_surface_type()
+  3. get_building_scc_skylight_roof_ratios()
+  4. data_lookup()
+
+## Rule Logic:  
+
+- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- Get building skylight roof ratios dictionary: `building_scc_skylight_roof_ratios_dictionary = get_building_scc_skylight_roof_ratios(B_RMR)`
+
+- For each building in the Baseline model: `for building_b in B_RMR.building:`
```

Comment:
small fix required to remove the check for each building.

Reply:
a high level applicability check to verify that RMR has a single building. If a RMR has multiple buildings then it should be flagged early on by the RCT and user needs to submit a modified RMR